test/test_transforms_v2.py

```diff
@@ -1071,25 +1143,27 @@ def test_image_correctness(self, fn):
 
         torch.testing.assert_close(actual, expected)
 
-    def _reference_horizontal_flip_bounding_boxes(self, bounding_boxes):
+    def _reference_horizontal_flip_bounding_boxes(self, bounding_boxes, format):
         affine_matrix = np.array(
             [
                 [-1, 0, bounding_boxes.canvas_size[1]],
                 [0, 1, 0],
             ],
         )
 
+        if tv_tensors.is_rotated_bounding_format(format):
+            return reference_affine_rotated_bounding_boxes_helper(bounding_boxes, affine_matrix=affine_matrix)
```

Comment:
Nit: it's personal but I find it clearer to have all the `return` statement with the same level of indentation. It makes it easier to find where the exit points are, which is important as functions get bigger. So here (and below) I would suggest either:

```py
if: ...
	return ...
else:
	return ...
```

or

```py
helper = ... if ... else ...
return helper(...)
```